src/utilities/introspectionQuery.js

```diff
@@ -273,4 +274,5 @@ export type IntrospectionDirective = {|
   +description?: ?string,
   +locations: $ReadOnlyArray<DirectiveLocationEnum>,
   +args: $ReadOnlyArray<IntrospectionInputValue>,
+  +repeatable: boolean,
```

Comment:
@OlegIlyenko Can you please address this comment in spec RFC:
https://github.com/facebook/graphql/pull/472/files/1ba4e196aa710a10eceaa346d02a970dfd0b2d3a#diff-35fb422eded6e8f4df638c0d159008f6

It blocks this PR but I think we should discuss it in the scope of spec RFC.

Reply:
Could you please describe in more detail what kind of information you would like to add? I'm not 100% sure I understood your comment?

Reply:
@OlegIlyenko I mean if merge this PR that will expose `repeatable` through introspection and it would be significantly harder to rename it later.
As I wrote in my comment to the spec RFC I think this field should be named `isRepeatable` for consistency with `isDeprecated`.

Everything else in this PR looks good so would be great to discuss this last point but since this PR is just an implementation of spec RFC I think we should discuss it in spec PR.

Reply:
I see, thanks for the clarification. Sure thing, I will open a discussion on this topic in the RFC PR.

Reply:
@OlegIlyenko I feel very stupid because in the first comment of this thread I linked review comment I forget to submit (review comments UI is very weird).
I already submitted it in.

Reply:
@OlegIlyenko Now it visible: https://github.com/facebook/graphql/pull/472/files/1ba4e196aa710a10eceaa346d02a970dfd0b2d3a#diff-35fb422eded6e8f4df638c0d159008f6

Reply:
👍 i see, thanks! now it's clear, i was confused about wrong link

Reply:
I did the renaming https://github.com/graphql/graphql-js/pull/1541/commits/56fee7d28b68d44665007e175129839af11ae8d2 I will adjust the spec text accordingly